env0/resource_environment_discovery_configuration.go

```diff
@@ -154,6 +154,24 @@ func resourceEnvironmentDiscoveryConfiguration() *schema.Resource {
 				Default:      false,
 				RequiredWith: []string{"token_id"},
 			},
+			"is_bitbucket_server": {
```

Comment:
Just wondering, I don't think it is a real problem, but once a user who has a env_discovery resource with previous provider version and then upgrade to the version containing this change, will they see a drift? (from not having this props as false and now having them)

Reply:
it's possible.
but generally there's a default: "false". So I believe if they match it wouldn't cause a drift.
if it's set to true in the in the backend - then it will probably cause a drift that users will need to fix and set to 'true' in the schema. (Which is expected).